docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2022-03-28
+ - Add support for absolute filepath in API Scan target option.
```

Comment:
```suggestion
- Add support for absolute filepath in API Scan target option.
```

Reply:
@kingthorin Looks like a space before a dash is in harmony with other entries in changelog file.